shared/templates/template_BASH_grub2_bootloader_argument

```diff
@@ -14,5 +14,7 @@ fi
 grubby --update-kernel=ALL --args="{{{ ARG_NAME_VALUE }}}"
 {{% else %}}
 # Correct grub2 kernelopts value using grub2-editenv
-grub2-editenv - set "$(grub2-editenv - list | grep kernelopts) {{{ ARG_NAME_VALUE }}}"
+if ! grub2-editenv - list | grep -qE '^kernelopts(=|=.*\s){{{ ARG_NAME_VALUE }}}($|\s).*$'; then
```

Comment:
Unless I miss something, I guess the regex can be simplified a bit:
`'^kernelopts=(.*\s)?{{{ ARG_NAME_VALUE }}}(\s.*)?$'`

Same thing for the Ansble template.